lib/engine/game/g_1832/step/token.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/token'
+
+module Engine
+  module Game
+    module G1832
+      module Step
+        class Token < Engine::Step::Token
+          def actions(entity)
+            actions = super
+            actions += %w[choose pass] if can_place_miami_token?(entity)
+
+            actions.uniq
```

Comment:
This can be made a tiny bit simpler, and avoid the need to call `uniq`, by using the array union operator.
```suggestion
            return super unless can_place_miami_token?(entity)

            super | %w[choose pass]
```